Libraries/PeriphDrivers/Include/MAX32570/cameraif.h

```diff
@@ -66,6 +66,16 @@ typedef enum {
     MXC_PCIF_DATAWIDTH_12_BIT, ///<
 } mxc_pcif_datawidth_t;
 
+/**
+ * @brief   The list of Camera GPIO Datawidth options supported
+ *
+ */
+typedef enum {
+    MXC_PCIF_GPIO_DATAWIDTH_8_BIT = 0, ///<
+    MXC_PCIF_GPIO_DATAWIDTH_10_BIT, ///<
+    MXC_PCIF_GPIO_DATAWIDTH_12_BIT, ///<
+} mxc_pcif_gpio_datawidth_t;
```

Comment:
I know the enum names are pretty much self-explanatory, but we could still add a few words to avoid having empty comment blocks.